app/celery/scheduled_tasks.py

```diff
@@ -189,3 +199,140 @@ def check_templated_letter_state():
                 message=msg,
                 ticket_type=zendesk_client.TYPE_INCIDENT
             )
+
+
+def _get_dynamodb_comp_pen_messages(table, message_limit: int) -> list:
+    """
+    Helper function to get the Comp and Pen data from our dynamodb cache table.
+
+    :param table: the dynamodb table to grab the data from
+    :param message_limit: the number of rows to search at a time and the max number of items that should be returned
+    :return: a list of entries from the table that have not been processed yet
+    """
+
+    results = table.scan(
+        FilterExpression=boto3.dynamodb.conditions.Attr('is_processed').eq(False),
+        Limit=message_limit
+    )
+
+    items: list = results.get('Items')
+
+    # Keep getting items from the table until we have the number we want to send, or run out of items
+    while 'LastEvaluatedKey' in results and len(items) < message_limit:
+        results = table.scan(
+            FilterExpression=boto3.dynamodb.conditions.Attr('is_processed').eq(False),
+            Limit=message_limit,
+            ExclusiveStartKey=results['LastEvaluatedKey']
+        )
+
+        items.extend(results['Items'])
+
+    return items[:message_limit]
+
+
+@notify_celery.task(name='send-scheduled-comp-and-pen-sms')
+@statsd(namespace='tasks')
+def send_scheduled_comp_and_pen_sms():
+    if not is_feature_enabled(FeatureFlag.COMP_AND_PEN_MESSAGES_ENABLED):
+        current_app.logger.warning('Attempted to run send_scheduled_comp_and_pen_sms task, but feature flag disabled.')
+        return
+
+    # this is the agreed upon message per minute limit
+    messages_per_min = 3000
+    dynamodb_table_name = os.getenv('COMP_AND_PEN_DYANMODB_NAME')
+    service_id = os.getenv('COMP_AND_PEN_SERVICE_ID')
+    template_id = os.getenv('COMP_AND_PEN_TEMPLATE_ID')
+
+    # TODO: utils #146 - Debug messages currently don't show up in cloudwatch, requires a configuration change
+    current_app.logger.debug('send_scheduled_comp_and_pen_sms connecting to dynamodb')
+
+    # connect to dynamodb table
+    dynamodb = boto3.resource('dynamodb')
+    table = dynamodb.Table(dynamodb_table_name)
+
+    # get messages to send
+    try:
+        comp_and_pen_messages: list = _get_dynamodb_comp_pen_messages(table, messages_per_min)
+    except Exception as e:
+        current_app.logger.error(
+            'Exception trying to scan dynamodb table for send_scheduled_comp_and_pen_sms exception_type: %s - '
+            'exception_message: %s', type(e), e
+        )
```

Comment:
This is a path-ending state. Let's make this one a critical.
```suggestion
        current_app.logger.critical(
            'Exception trying to scan dynamodb table for send_scheduled_comp_and_pen_sms exception_type: %s - '
            'exception_message: %s', type(e), e
        )
```